Assets/Scripts/Cutscene.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Cutscene: MonoBehaviour
+{
+	private static float TextBubbleWidth = 128.0f;
+	private GameObject player = null;
+	public Text textObject;
+	private bool running = false;
+	public string[] textBubbles;
+	public GameObject[] objects;
+	public int[] textBubbleOwners;

```

Comment:
I think there could be an easier way to set this up for the editor. I did a similar mapping in the AudioClipPlayer script - basically you create a serializable struct with two values, and make a public array of it. That way you don't have to have three public objects for the text, the objects, and correlating the two together. In this case the struct would have a GameObject and a string for that object to say.